perbaikan kecil untuk login form

totoprayogo1916 opened the pull request:
- attribute `required` harusnya ada diluar class,
- tambah '<label/>' untuk checkbox, jadi text pada checkbox bisa diklik

<!-- 
Solusi untuk {perbaikan|fitur baru} terkait issue {#1, #2, #3, dst.}

Cek : 
1. [Aturan Penulisan Script.](https://github.com/OpenSID/OpenSID/wiki/Aturan-Penulisan-Script)
2. [Proses Review Pull Request.](https://github.com/OpenSID/OpenSID/wiki/proses-review-pull-request)
-->

totoprayogo1916 commented:
ngomong-ngomong,
branch mana ya yang dikembangkan?

waktu kirim PR ini diminta (wiki) targetnya adalah branch Master ..

andifahruddinakas commented:
> ngomong-ngomong, branch mana ya yang dikembangkan?
> 
> waktu kirim PR ini diminta (wiki) targetnya adalah branch Master ..

buat branch untuk PR dari dev-21.12 dan targetkan ke branch dev-21.12

totoprayogo1916 commented:
baik, saya rebase dulu untuk perubahannya agar bisa ke branch dev-*

Commit:
perbaikan kolom pada laman login
- attribute `required` harusnya ada diluar class
- penambahan tag `label` untuk checkbox

totoprayogo1916 commented:
sudah ya @andifahruddinakas
